### DISCUSSION TITLE
Multiple locales on same site

### DISCUSSION THREAD

**Discussion by freeform99** [umputun/remark42#1295] [Q&A] [answered]
Hi there - thanks for this great software. 

I'd like to try to use it on a site with multiple locales. Is it possible to have the comments displayed in (say) German on one page and in French on another? 

I tried to include a configuration object on two different pages (with different `locale` variable) but that doesn't seem to work. I wonder if it is my mistake (and what the right way would be) or a limitation of remark42.

Thank you!

**Comment by freeform99** [comment, marked as answer]
Found the solution myself - it works! I am using Svelte/SvelteKit and was able to use the Gatsby instructions, with some small modifications. I can pass the language into the component as a prop and the corresponding version of the remark42 comments field is loaded.

**Reply by paskal** [reply]
Hey, would be great if you'll share your findings somewhere [in the doc](https://remark42.com/docs/configuration/frontend/) so it would be easier for others to do the same if they need to.
